udemy_exercises/cell_phone/Main.java

```diff
@@ -0,0 +1,89 @@
+import java.util.Scanner;
+
+public class Main {
+
+    private static Scanner scanner = new Scanner(System.in);
+    private static Cellphone myPhone = new Cellphone(1234567890);
+
+    public static void main(String[] args) {
+        boolean active = true;
+        Options();
+        while (active) {
+            int action = scanner.nextInt();
```

Comment:
The int 'action' is instantiated on every loop of the 'while' block.  How might this be refactored to instantiate 'action' only once?